### PR TITLE
Add questions about dropzone logic in `repo-legacy.js`

### PR DESCRIPTION
**Pull request by yardenshoham**
While working on removing jQuery AJAX calls from this file I couldn't find a way to trigger any AJAX calls related to the dropzone. It honestly seems to me like dead code, but I have no historical context regarding this logic so I don't know if it's ok to delete it altogether.

### CODE REVIEW

**Review by silverwind — APPROVED** [review]
Should remove dropzone eventually :)

File: web_src/js/features/repo-legacy.js [hunk 1]
```diff
@@ -442,11 +442,17 @@ async function onEditContent(event) {
     comboMarkdownEditor = await initComboMarkdownEditor($editContentZone.find('.combo-markdown-editor'));
 
     const $dropzone = $editContentZone.find('.dropzone');
+
+    // is setupDropzone doing anything? We already have have initGlobalDropzone to control all dropzones
```
Comment (wxiaoguang) [inline web_src/js/features/repo-legacy.js:454]: No. This is for the `$editContentZone.html($('#issue-comment-editor-template').html());` above, L441

File: web_src/js/features/repo-legacy.js [hunk 2]
```diff
@@ -442,11 +442,17 @@ async function onEditContent(event) {
...
     const dz = await setupDropzone($dropzone);
+
+    // there is no such button, why is this code here?
```
Comment (wxiaoguang) [inline web_src/js/features/repo-legacy.js:457]: They are from `#issue-comment-editor-template`

**Comment by yardenshoham** [comment]
@wxiaoguang not sure I fully understand. How can I trigger the AJAX calls in this file?

**Comment by wxiaoguang** [comment]
> @wxiaoguang not sure I fully understand. How can I trigger the AJAX calls in this file?

IIRC, edit a comment

**Comment by yardenshoham** [comment]
Thanks I will send a PR

**Pull request closed without merging**